spec/models/ems_event_spec.rb

```diff
@@ -102,6 +102,62 @@ def mock_raw_event_vm(raw_event)
     end
   end
 
+  context ".process_container_entities_in_event!" do
+    before :each do
+      @ems = FactoryGirl.create(:ems_kubernetes)
+      @container_project = FactoryGirl.create(:container_project, :ext_management_system => @ems)
+      EMS_REF = "test_ems_ref"
```

Comment:
@dkorn For future reference, don't use constants in RSpec...they don't play nice and cause deprecation warnings, because they get redefined on every run.  In this case a simple instance variable or a `let` would work.

cc @kbrock